### PR TITLE
Change time format to Norwegian

### PR DESCRIPTION
**Pull request by EinarUeland**

### CODE REVIEW

File: frontend/src/components/MissionOverview/UpcomingMissionCard.tsx [hunk 1]
```diff
@@ -113,7 +114,13 @@ function MissionStartTimeDisplay({ mission }: MissionDisplayProps) {
     return (
         <StyledMissionStartTime>
             <Typography>Date: {format(new Date(mission.startTime), 'dd. MMM')}</Typography>
-            <Typography>Time: {format(new Date(mission.startTime), 'hh:mm')}</Typography>
+            <Typography>
+                Time:{' '}
+                {new Date(mission.startTime).toLocaleTimeString('no-NO', {
```
Comment (EinarUeland) [inline frontend/src/components/MissionOverview/UpcomingMissionCard.tsx:119]: This could also use the "default time", but chose to use Norwegian

**Review by GodVenn — COMMENTED** [review]
Is it possible to get the timezone based on where the frontend is being called from, i.e. where the user is located?
Or maybe the timezone should be based on the location of the plant?
This might make sense to change later when we get more info from the fusion context, maybe they have timezones.

**Comment by EinarUeland** [comment]
> Is it possible to get the timezone based on where the frontend is being called from, i.e. where the user is located? Or maybe the timezone should be based on the location of the plant? This might make sense to change later when we get more info from the fusion context, maybe they have timezones.

File: frontend/src/components/MissionOverview/UpcomingMissionCard.tsx (hunk 1 above)
Comment (GodVenn) [reply frontend/src/components/MissionOverview/UpcomingMissionCard.tsx:119]: Good choice

**Comment by EinarUeland** [comment]
> Is it possible to get the timezone based on where the frontend is being called from, i.e. where the user is located? Or maybe the timezone should be based on the location of the plant? This might make sense to change later when we get more info from the fusion context, maybe they have timezones.

The default is to use local clock. In this case I have specified "Norwegian time".  There are pros and cons to specifying time-zone or letting the users system determine it. For now I think using Norwegian makes sense.

Also - I just checked another Equinor app. Here the correct time is displayed in the planning window though a digital clock. I think this is a decent method to avoid all clock errors and is something we might consider for later.

I'm also noting that currently the time-field does not give much sense as it just displays the time the mission was created in.

**Comment by aestene** [comment]
This pull request has automatically been marked as stale as there has been no activity for 30 days. Remove stale label or comment or this will be closed in 7 days.

**Pull request closed without merging**